File: content.r
con_in <- file("input.txt", "r")
con_out <- file("output.txt", "w")

while (length(line <- readLines(con_in, n = 1, warn = FALSE)) > 0) {
  writeLines(toupper(line), con_out)
}

close(con_in)
close(con_out)

File: content.java
import java.io.*;

public class UpperCaseConverter {
    public static void main(String[] args) throws IOException {
        BufferedReader reader = new BufferedReader(new FileReader("input.txt"));
        BufferedWriter writer = new BufferedWriter(new FileWriter("output.txt"));
        String line;

        while ((line = reader.readLine()) != null) {
            writer.write(line.toUpperCase());
            writer.newLine();
        }

        reader.close();
        writer.close();
    }
}
